src/iris/ui/static/js/iris.js

```diff
@@ -1990,6 +1993,14 @@ iris = {
         toggleReprioritization: function() {
             $('#reprioritization').toggleClass('active')
         },
+        getModes: function(){
```

Comment:
You might be able to use `window.appData.modes` to get at the list of modes, which would be a lot faster than adding another ajax call here.

Reply:
Good call, changed